src/components/tabs/tab/tab.d.ts

```diff
@@ -1,13 +1,19 @@
-import * as React from 'react';
+import * as React from "react";
 import { PaddingSpacingProps } from "../../../utils/helpers/options-helper";
 
+export interface TabContextProps {
+  setError: (childId: string, hasError: boolean) => void;
+  setWarning: (childId: string, hasWarning: boolean) => void;
+  setInfo: (childId: string, hasInfo: boolean) => void;
+}
+
 export interface TabProps extends PaddingSpacingProps {
   title?: string;
   tabId: string;
   className?: string;
   children?: React.ReactNode;
   isTabSelected?: boolean;
-  position: 'top' | 'left';
+  position: "top" | "left";
```

Comment:
This should be optional

Reply:
looks like I missed that component, I also need to add documentation here

Reply:
This comment is still outstanding I think